recommendation/templates/recommendation/recommendation_rating.html

```diff
@@ -0,0 +1,189 @@
+    <h3>Teacher Recommendation for <mark><u>{{teacher_recommendation.user.first_name}} {{teacher_recommendation.user.last_name}}</u></mark></h3>
+    <h4>Name of Reviewer: {{teacher_recommendation.first_name}} {{teacher_recommendation.last_name}}</h4>
+    <form action="{% url 'recommendation_rating' teacher_recommendation.pk teacher_recommendation.user.pk %}" method="post" class="form-signup" style="padding-top: 10px;">
+        {% csrf_token %}
+        <div class="rounded bg-secondary" style="height:35px;margin-bottom:10px;">
+            <h5 class="p-2">I. Knowledge of Applicant</h5>
+        </div>
+        <div class="form-row">
+            <div class="form-label-group">
+                <span>Approximately how long have you known the applicant? (In years)</span> {{form.known_length}}
+                <br>
+            </div>
+        </div>
+        <div class="form-row">
+            <div class="form-label-group">
+                <span>How well do you feel like you know the applicant?</span><br>
+                <input type="radio" name="known_strength" value=0 required> Casually<br>
+                <input type="radio" name="known_strength" value=1> Well<br>
+                <input type="radio" name="known_strength" value=2> Very Well<br>
+                <br>
+            </div>
+
+        </div>
+        <div class="form-row">
+            <div class="form-label-group">
+                <span>What was the nature of your contact with the applicant?</span><br>
+                <input type="radio" name="known_location" value=0 required> Teacher in One Class<br>
+                <input type="radio" name="known_location" value=1> Teacher in Multiple Classes<br>
+                <input type="radio" name="known_location" value=2> Advisor<br>
+                <input type="radio" name="known_location" value=3> Mentor<br>
+                <input type="radio" name="known_location" value=4> Other<br>
+                <br>
+            </div>
+        </div>
+        <div class="rounded bg-secondary" style="height:35px;margin-bottom:10px;margin-top:15px;">
+            <h5 class="p-2">II. Relative Rating of the Applicant</h5>
+        </div>
+        <div class="form-row">
+            <p>Please rate the applicant in the areas indicated below by comparing him or her to a reference group of other students. </p>
+        </div>
+        <div class="form-row">
+            <table>
+                <tr>
+                    <th></th>
+                    <th>Top 1-2%</th>
+                    <th>Top 5%</th>
+                    <th>Top 10%</th>
+                    <th>Top 25%</th>
+                    <th>2nd 25%</th>
+                    <th>3rd 25%</th>
+                    <th>Last 25%</th>
+                    <th>Unable To Rate</th>
+                </tr>
+                <tr>
+                    <th> Ability to grasp new concepts </th>
+                    <th><input type="radio" name="rating_concepts" value=1 required></th>
```

Comment:
Nit - 
A bit concise way to get the same functionality working would be to have a {% forloop %} in the range for the number of elements, since the only part differing is the value field (which can be substituted by {{ forloop.counter }}.

Reply:
would this not require additional information to be passed through as context?